package.json

```diff
@@ -56,22 +58,49 @@
     "@babel/plugin-transform-typescript": "7.15.8",
     "@babel/preset-env": "7.15.8",
     "@babel/register": "7.15.3",
+    "@docusaurus/core": "2.0.0-beta.9",
+    "@docusaurus/preset-classic": "2.0.0-beta.9",
+    "@mdx-js/react": "^1.6.22",
+    "@svgr/webpack": "^5.5.0",
     "@types/chai": "4.2.22",
     "@types/mocha": "9.0.0",
     "@types/node": "16.11.1",
     "@typescript-eslint/eslint-plugin": "5.0.0",
     "@typescript-eslint/parser": "5.0.0",
+    "babel-eslint": "^10.1.0",
     "chai": "4.3.4",
+    "clsx": "^1.1.1",
     "cspell": "5.12.3",
+    "docusaurus-plugin-typedoc-api": "^1.6.1",
     "eslint": "7.32.0",
     "eslint-plugin-import": "2.25.2",
     "eslint-plugin-internal-rules": "file:./resources/eslint-internal-rules",
     "eslint-plugin-istanbul": "0.1.2",
     "eslint-plugin-node": "11.1.0",
+    "eslint-plugin-react": "^7.27.0",
+    "eslint-plugin-react-hooks": "^4.3.0",
     "eslint-plugin-tsdoc": "0.2.14",
+    "file-loader": "^6.2.0",
     "mocha": "9.1.3",
     "nyc": "15.1.0",
     "prettier": "2.4.1",
-    "typescript": "4.4.4"
+    "prism-react-renderer": "^1.2.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "typedoc": "^0.22.8",
+    "typescript": "4.4.4",
+    "url-loader": "^4.1.1"
+  },
+  "browserslist": {
+    "production": [
+      ">0.5%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
```

Comment:
Can we move it to `docs/.babelrc` (preferred solution) or browserslist specific file?
In general, we don't keep configurations in `package.json` unless it the only place they can leave?